ext/hash/hash.stub.php

```diff
@@ -2,37 +2,37 @@
 
 /** @generate-function-entries */
 
-function hash(string $algo, string $data, bool $raw_output = false): string|false {}
+function hash(string $algorithm, string $data, bool $binary_output = false): string|false {}
 
-function hash_file(string $algo, string $filename, bool $raw_output = false): string|false {}
+function hash_file(string $algorithm, string $filename, bool $binary_output = false): string|false {}
 
-function hash_hmac(string $algo, string $data, string $key, bool $raw_output = false): string|false {}
+function hash_hmac(string $algorithm, string $data, string $key, bool $binary_output = false): string|false {}
 
-function hash_hmac_file(string $algo, string $data, string $key, bool $raw_output = false): string|false {}
+function hash_hmac_file(string $algorithm, string $data, string $shared_secret_key, bool $binary_output = false): string|false {}
```

Comment:
```suggestion
function hash_hmac_file(string $algorithm, string $data, string $key, bool $binary_output = false): string|false {}
```
Keys are always secret, and I'm not sure why there is a "shared" here (shared with whom? Whether this key is shared depends on the usage.)

This is also inconsistent with hash_hmac() above.